README.md

```diff
@@ -16,84 +16,84 @@ We should be as specific and thorough as possible when defining our style, testi
 
 #### 1. Names of internal functions in a library should not have an underscore prefix.
 
-The style guide states
-
-> Underscore Prefix for Non-external Functions and Variables
-
-One of the motivations for this rule is that it is a helpful visual clue.
+YES:
 
-> Leading underscores allow you to immediately recognize the intent of such functions...
+```solidity
+Library.function()
+```
 
-We agree that a leading underscore is a useful visual clue, and this is why we oppose using them for internal library functions that can be called from other contracts. Visually, it looks wrong.
+NO:
 
 ```solidity
 Library._function()
 ```
 
-or
+If a function should never be called from another contract, it should be marked private and its name should have a leading underscore.
+
+#### Justification
```

Comment:
We don't have `Justification` elsewhere where we explain reasoning, so I am hesitant to mix up the format. (I used to have something like this but removed for consistency)

Reply:
I think exceptions should require justification. This is the only exception we've listed so I would expect future ones to have justification as well, but might not be present right now